.github/workflows/.windows.yml

```diff
@@ -246,14 +269,17 @@ jobs:
           echo ${{ steps.tests.outputs.matrix }}
 
   integration-test:
+    if: ${{ matrix.runtime != 'builtin' && matrix.storage != 'snapshotter' }}
```

Comment:
Should that be `||`? I'm guessing this is supposed to be excluding `snapshotter+builtin`.

Seems easier to add an exclude stanza to the strategy to me.